apps/docs/src/.vitepress/utils-icons.ts

```diff
@@ -0,0 +1,33 @@
+import { ICON_CATEGORIES } from "@sit-onyx/icons";
+import { capitalize } from "vue";
+
+const getIconContextData = (iconName: string, ALL_ICONS: Record<string, string>) => {
+  const parts = iconName.split("-");
+  // bell-disabled => `Bell Disabled`
+  const tooltipName = parts.map((word) => capitalize(word)).join(" ");
+  // bell-disabled => `bellDisabled`
+  const importName = parts
```

Comment:
info: the importName is not used yet, it will be used in the copy functionality which will be added in the next PR.